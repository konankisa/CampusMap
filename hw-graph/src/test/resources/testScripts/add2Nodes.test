# Creates an empty graph and adds 2 nodes to it

# Create a graph
CreateGraph graph

# Add a node to graph
AddNode graph node1

# Add another node to graph
AddNode graph node2