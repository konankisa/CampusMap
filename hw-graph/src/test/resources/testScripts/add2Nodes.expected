# Creates an empty graph and adds 2 nodes to it

# Create a graph
created graph graph

# Add a node to graph
added node node1 to graph

# Add another node to graph
added node node2 to graph